### DISCUSSION TITLE
Getting error Could not parse body - request body did not fit into client body buffer, consider raising 'client_body_buffer_size'

### DISCUSSION THREAD

**Discussion by akulmehta** [prism-php/prism#112] [Q&A]
I was trying to do a query using Mistral's pixtral on 4 photos with a fairly large prompt and got the error. Does anyone know how to raise the client_body_buffer_size?

**Comment by pushpak1300** [comment]
Needs more info. 
Can you share what's your current configuration.  (PHP version, ini settings)
How big prompt you are trying to execute. ?

What is the iamge size and all.

**Reply by akulmehta** [reply]
Here is some more information:
1. PHP Version: 8.3.12 
2. What ini settings do you want? here are some related to memory:
```
PHP_MEMORY_LIMIT 	256M
PHP_POST_MAX_SIZE 	100M
PHP_UPLOAD_MAX_FILE_SIZE 	100M
```

The token size of the prompt is roughly 15,000 tokens (including the images). When I have 3 images the prompt is approx 11,000 tokens and with 4 images it is roughly 15,000 tokens. The images are <500 kb (roughly 1536x2048). This error appears only when I use 4 images  in a prompt and only with certain sets of images.
